### DISCUSSION TITLE
Double Dash L/R functionality on GC Blue Retro

### DISCUSSION THREAD

**Discussion by Diloret2** [darthcloud/BlueRetro#516] [Q&A]
Hello!  I recently got, and installed the Laserbear GC Blue Retro into my Pico Boot/Swiss GameCube.  So far, everything appears to work well, connectivity issues here and there, but my biggest thing I've discovered is only in Double dash, the L/R don't work in the race, so I can't drift.  L/R works fine in F-Zero, for example.  I updated to 1.8.1, my Swiss is the latest update and tried the merge digital/analog preset, even tried an Xbox Series X and PS5 controller, but still the same result.  Is there something else I can try, or another setting?

**Comment by darthcloud** [comment]
Did you program the merged trigger presets for having both analog and digital trigger map to the PS analog only trigger?
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#33---example-preset-load

**Reply by Diloret2** [reply]
I did do that.

**Reply by darthcloud** [reply]
Sorry I didn't read until the end.

I don't have access to my Cube to test first hand. If the drift is done solely by press the analog part (not going all the way to the digital "clicky" part) you need to avoid pressing the trigger all the way in as you don't have a tactile feedback for when the transition occur.

On the other hand if you do need to press all the way to clicky part on an OG controller maybe try reducing the threshold at which the digital part start:
![triggers_tweak](https://user-images.githubusercontent.com/3744056/209399948-f07563a6-fe19-4143-a9c0-42cfee34107e.gif)

**Reply by Diloret2** [reply]
Ok, perfect!  I will try and give this a shot, thanks for the help!
